packages/common/src/initiatives/_internal/computeProps.ts

```diff
@@ -43,6 +43,7 @@ export function computeProps(
       model.data.view.featuredImageUrl,
       requestOptions
     ),
+    featuredImageAltText: model.data.view.featuredImageAltText,
```

Comment:
hmm, it seems to me that this should already be mapped over by the property mapper...and same with the hero property actually. I think the real issue here might be that we're overwriting `initiative.view` rather than updating the properties that actually need "computing". Does it work if you just have the following instead: 

`initiative.view = {
  ...model.data.view,
  featuredImageUrl: getAuthedImageUrl(
    model.data.view.featuredImageUrl,
    requestOptions
  )
}

Reply:
You're right, we can get it from the model data